cmd/pool/account.go

```diff
@@ -172,30 +199,42 @@ func newAccount(ctx *cli.Context) error {
 	return nil
 }
 
-func printAccountFees(client poolrpc.TraderClient, amt btcutil.Amount,
-	confTarget uint32) error {
+func printAccountFees(
```

Comment:
nit: we don't usually use this more JS type formatting but instead use the full line width for the function header.
Also, after a multi-line function definition, we add a newline to better show where the body begins.